src/autopas/containers/verletClusterLists/VerletClusterLists.h

```diff
@@ -738,7 +739,7 @@ class VerletClusterLists : public ParticleContainerInterface<FullParticleCell<Pa
     auto towersOfInterestIterator = towersOfInterest.begin();
     for (size_t i = 0; i < towersOfInterstPerDim[1]; ++i) {
       std::iota(towersOfInterestIterator, towersOfInterestIterator + towersOfInterstPerDim[0],
-                firstTowerIndex + i * _towersPerDim[0]);
+                std::min(firstTowerIndex, lastTowerIndex) + i * _towersPerDim[0]);
```

Comment:
jo, das sieht besser aus :)

Reply:
btw. such accesses (e.g. `getRegioniterator({10,10,10},{5,5,5})` are never tested and will probably lead to a lot of problems...

Reply:
-> #536